backend/dataall/modules/common/sagemaker/permissions.py

```diff
@@ -0,0 +1,27 @@
+"""
```

Comment:
In an ideal world, permissions should be module specific like we already have notebook's permissions in `dataall.modules.notebooks.services.permissions.py`.
But we also have a set of permissions that are shared between notebooks and ML studio. That was the primary reason why they are in `common`. Plus under the hood, ML studio uses the same policy as notebooks

`common` exists to avoid the code duplication between modules. Though ML studio is not a module yet.

At the moment `common` has a flat structure, since we expect that the modules will be pretty isolated and there will be not much intersections. But if we see a lot of `common` code we can create a deeper structure by creating of submodules  under `common` . Each submodule will represent a specific code share like `sagemaker` where we put the policy and the permissions between notebooks and ML studio there, or `shares` to put code for duplication of different share mechanisms, and etc

Reply:
> But we also have a set of permissions that are shared between notebooks and ML studio
Does this mean if I import both ML studio and notebooks and grant notebooks permissions, I will automatically gain some permissions for ML studio resources? Shouldn't these be separated?

Reply:
I also think each module should have separated permissions (idk if I already commented it). Because even if ML Studio and Notebooks share some IAM permissions, there will be other that they don't and for a case where only Notebooks are enabled, we would be granting additional permissions for ML Studio that are not needed

Reply:
- [x] Created more granular permissions